### ISSUE TITLE
Duplicate use of the description field in uom-plugin.cabal.

### ISSUE THREAD

**Issue by philderbeast** [adamgundry/uom-plugin#35]
When using `cabal new-build` I see duplicate use of the description field reported.

```
uom-plugin/uom-plugin.cabal:5:1:
The field "description" is specified more than once at positions 5:1, 20:1
```

**Comment by philderbeast** [comment]
The same warning is reported when building with stack;

```
uom-plugin> stack build

Warning: Cabal file warning in
         .../uom-plugin/uom-plugin.cabal@ 5:1: The field
         "description" is specified more than once at positions 5:1, 20:1
```

**Comment by adamgundry** [comment]
Resolved by eb1761a
